dependencies/qr_deps.py

```diff
@@ -42,3 +45,48 @@ def generate_url_qr(qr_key: str, user: user_to_register) -> str:
     return url_qr
 
 
+def get_value_key_qr(key_qr: str):
+    """
+        Get value of key_qr
+
+        Parameters
+        ----------
+        key_qr: str
+                string obtained by the client
```

Comment:
Remember to start with CAPITAL LETTERS at the beginning of new paragraphs for documentation